src/content/docs/ai-monitoring/intro-to-ai-monitoring.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Introduction to AI monitoring'
+metaDescription: 'AI monitoring lets you observe the AI-layer of your tech stack, giving you a holistic overview of the health and performance of your AI-powered app.'
+freshnessValidatedDate: never
+---
+
+import aiTraceViewIntroPage from 'images/ai_screenshot-full_trace-view-intro-page.webp'
+
+import aiAIResponsesOverview from 'images/ai_screenshot-full_AI-responses-overview.webp'
+
+When New Relic talks about AI, we're referring to the layer of your environment that uses a large language model (LLM) to generate a response when it receives an end user prompt. Your app contains an AI assistant, but behind the AI are libraries, cost implications, vector databases, and new processes that can be monitored.   
+
+<img
```

Comment:
As much as I like the image, would it be possible to get a screenshot with an AI query that might be a bit more relevant to a real life user case?